static/js/tippyjs.js

```diff
@@ -8,6 +8,16 @@ import * as reactions from "./reactions";
 import * as rows from "./rows";
 import * as timerender from "./timerender";
 
+function get_tooltip_content(reference) {
+    if ("tooltipId" in reference.dataset) {
+        const template = document.querySelector(`#${CSS.escape(reference.dataset.tooltipId)}`);
+        return template === null
+            ? `No tooltip for ${reference.dataset.tooltipId}`
```

Comment:
Maybe "Could not find tooltip template with ID {reference.dataset.tooltipId}."

Reply:
Oh yeah, this was a temporary debugging message. I’d actually be fine with letting this throw an error—we’re inside Tippy’s `mouseenter` handler so it’s not going to interfere with anything else.